scaling_performance/host_practices.adoc

```diff
@@ -39,6 +39,21 @@ kubernetesMasterConfig:
     - "1000"
 ----
 
+For large and/or dense clusters, the apiserver might get overloaded because of the low 
+default limits. Provided sufficient CPU and memory resources are available on the 
+apiserver system(s), this issue can be safely alleviated by doubling or quadrupling the 
+default api-qps and api-burst values in the *_/etc/origin/master/master-config.yaml_* file:
```

Comment:
Believe these need to be changed in the node-config-master configmap, right?

Reply:
> Believe these need to be changed in the node-config-master configmap, right?

It depends on which on you mean. The master loops back to itself, so if you ran into rate limiting with the loopback connection, this may be the right spot.